### DISCUSSION TITLE
Doris倒排索引如何按照指定字符分词

### DISCUSSION THREAD

**Discussion by icesong** [apache/doris#38963] [A - General / Q&A]
在Doris 2.1版本中倒排索引如何按照指定字符进行进行分词？ 
例如：`北京,上海,深圳的同学`，按照英文逗号分词为：`北京` `上海` `深圳的同学`

**Comment by maruyue** [comment]
建议英文提问，如使用中文提问，可以去ask.selectdb.com中文技术论坛。
